src/aluno/CadastroAlunoPage.tsx

```
@@ -0,0 +1,304 @@
+                                    </div>
+
+                                    {/* Nome */}
+                                    <div className="flex flex-col items-start gap-1">
+                                        <span className="text-sm text-black">Nome Completo</span>
+                                        <FormField
+                                            control={form.control}
+                                            name="nomeCompleto"
+                                            render={({ field }) => (
+                                                <FormItem className="w-full">
+                                                    <FormControl>
+                                                        <Input
```

ESLint

[error] Replace `························································` with `····························`

(prettier/prettier)